modules/admin-top-bar/module.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
 namespace Elementor\Modules\AdminTopBar;
```

Comment:
There is leftover code in JS and maybe in CSS. After talking with Aviad, he decided to open another task for that and merge this code as fast as we can.